packages/mermaid/src/diagrams/quadrant-chart/quadrantRenderer.ts

```diff
@@ -0,0 +1,144 @@
+// @ts-ignore: TODO Fix ts errors
+import { select } from 'd3';
+import * as configApi from '../../config.js';
+import { log } from '../../logger.js';
+
+import { configureSvgSize } from '../../setupGraphViewbox.js';
+import { Diagram } from '../../Diagram.js';
+import {
+  QuadrantBuildType,
+  QuadrantPointType,
+  QuadrantQuadrantsType,
+  QuadrantTextType,
+  TextHorizontalPos,
+  TextVerticalPos,
+} from './quadrantBuilder.js';
+
+export const draw = (txt: string, id: string, _version: string, diagObj: Diagram) => {
```

Comment:
Does this draw the title if it's present?

Reply:
I missed it , added it.